changelog/unreleased/issue-5089

```diff
@@ -0,0 +1,22 @@
+Enhancement: Allow including or excluding extended file attributes
```

Comment:
The summary must be a single line shorter than 80 characters and not include trailing punctuation.

`Enhancement: Allow including/excluding extended file attributes during restore` would work even though it's a bit of cheating.